great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -237,6 +241,18 @@ def __init__(self, runtime_environment: Optional[dict] = None) -> None:
 
         # NOTE - 20210112 - Alex Sherstinsky - Validation Operators are planned to be deprecated.
         self.validation_operators: dict = {}
+        if (
+            "validation_operators" in self.get_config().commented_map  # type: ignore[union-attr]
+            and self.config.validation_operators
+        ):
+            for (
+                validation_operator_name,
+                validation_operator_config,
+            ) in self.config.validation_operators.items():
+                self.add_validation_operator(
+                    validation_operator_name,
+                    validation_operator_config,
+                )
```

Comment:
Ported over from `BaseDataContext` - required for proper validation operator porting